communication/slack-config/sig-release/usergroups.yaml

```diff
@@ -6,6 +6,11 @@ usergroups:
     description: Release Managers. Ping for questions on branch cuts and building/packaging Kubernetes.
     channels:
       - release-ci-signal
+      - release-comms
```

Comment:
Remove these. Release Managers don't need to be in these channels.

Reply:
addressed